src/main/java/org/folio/dao/impl/FileDefinitionDaoImpl.java

```diff
@@ -41,6 +52,19 @@ public FileDefinitionDaoImpl(@Autowired PostgresClientFactory pgClientFactory) {
       .map(fileDefinitions -> fileDefinitions.isEmpty() ? Optional.empty() : Optional.of(fileDefinitions.get(0)));
   }
 
+  @Override
+  public Future<List<FileDefinition>> getExpired(Date lastUpdateDate, String tenantId) {
```

Comment:
can we name it a little readable, like getExpiredEntries or getExpiredFileDefinitions etc..